### DISCUSSION TITLE
Mixed Chinese and English translation

### DISCUSSION THREAD

**Discussion by nomyfan** [community/community#38448] [Mobile]
![Screenshot_2022-11-07-09-25-56-863_com.github.android.jpg](https://user-images.githubusercontent.com/19922814/200208931-a491bd15-81b1-47bf-aad3-20a4571c894e.jpg)

I have updated to latest version(1.88.0).

This issue exists for a long time on my device.

My device info
Redmi K50, MIUI 13.0.24, zh-cn system language, English app language.

**Comment by aconsuegra** [comment]
Hi @nomyfan, thanks for getting in touch! I will create an issue for our engineers so they can investigate the issue.
